Dockerfile.cirun

```diff
@@ -1,7 +1,15 @@
 FROM quay-ci-base
+
+ENV ENCRYPTED_ROBOT_TOKEN_MIGRATION_PHASE remove-old-fields
```

Comment:
Not needed for now.

Also, do we still need the cirun file?

Reply:
Don't think so with tox. Removed the file.